hwilib/commands.py

```diff
@@ -282,9 +290,12 @@ def process_commands(args):
     getkeypool_parser.set_defaults(func=getkeypool)
 
     displayaddr_parser = subparsers.add_parser('displayaddress', help='Display an address')
-    displayaddr_parser.add_argument('path', help='The BIP 32 derivation path of the key embedded in the address')
-    displayaddr_parser.add_argument('--sh_wpkh', action='store_true', help='Display the p2sh-nested segwit address associated with this key path')
-    displayaddr_parser.add_argument('--wpkh', action='store_true', help='Display the bech32 version of the address associated with this key path')
+    group = displayaddr_parser.add_mutually_exclusive_group()
+    group.add_argument('--desc', help='Pseudo-descriptor for which to return a descriptor with keys, e.g. wpkh(00000000/84h/1h/0h/0/*). Alternatively, use --path with --sh_wpkh / --wpkh')
```

Comment:
what is "psuedo" about the option?

Reply:
I'm calling it a pseudo-descriptor, because there's no public key in it. The goal of a pseudo-descriptor like `wpkh(00000000/84h/1h/0h/0/*)` is to ask for an actual descriptor ``wpkh(00000000/84h/1h/0h/tpub..../0/*)` with a public key inserted in the right place. That's not covered in @sipa's document: https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md

Reply:
On further thought, I think these are ok. Might want to call them something like "descriptor requests" or something, and say exactly what info must reside in the requests, and what's missing.